processor/stream/test_approved_es_documents/testIntakeRUMV3Errors.approved.json

```diff
@@ -1,111 +1,44 @@
-                                "line": {
-                                    "column": 181,
-                                    "number": 1
-                                }
-                            },
-                            {
-                                "abs_path": "http://localhost:8000/test/e2e/general-usecase/bundle.js.map",
-                                "exclude_from_grouping": false,
-                                "filename": "~/test/e2e/general-usecase/bundle.js.map",
-                                "function": "runTask",
-                                "line": {
-                                    "column": 15,
-                                    "number": 1
-                                }
-                            },
-                            {
-                                "abs_path": "http://localhost:8000/test/e2e/general-usecase/bundle.js.map",
-                                "exclude_from_grouping": false,
-                                "filename": "~/test/e2e/general-usecase/bundle.js.map",
-                                "function": "invoke",
-                                "line": {
-                                    "column": 199,
-                                    "number": 1
-                                }
-                            },
-                            {
-                                "abs_path": "http://localhost:8000/test/e2e/general-usecase/bundle.js.map",
-                                "exclude_from_grouping": false,
-                                "filename": "~/test/e2e/general-usecase/bundle.js.map",
-                                "function": "timer",
-                                "line": {
-                                    "column": 33,
-                                    "number": 1
-                                }
-                            }
-                        ],
-                        "type": "Error"
-                    }
-                ],
-                "grouping_key": "52fbc9c2d1a61bf905b4a11c708006fd",
-                "id": "aba2688e033848ce9c4e4005f1caa534",
-                "log": {
-                    "message": "Uncaught Error: log timeout test error",
-                    "stacktrace": [
-                        {
-                            "abs_path": "http://localhost:8000/test/e2e/general-usecase/bundle.js.map",
-                            "exclude_from_grouping": false,
-                            "filename": "~/test/e2e/general-usecase/bundle.js.map",
-                            "function": "\u003canonymous\u003e",
-                            "line": {
-                                "column": 18,
-                                "number": 1
-                            }
-                        }
-                    ]
+                "culprit": "test/e2e/general-usecase/app.e2e-bundle.min.js?token=secret",
+                "custom": {
+                    "testContext": "testContext"
```

Comment:
Where did the `stacktrace` get lost? Looking at https://github.com/elastic/apm-server/pull/3414/files#diff-ed8e5b35bb28351f66974257d6d24873R2 one of the errors sends a stacktrace.

Reply:
great catch!